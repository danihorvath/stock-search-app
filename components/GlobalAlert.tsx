import React from "react";

const GlobalAlert: React.FC = () => {
  return (
    <div className="bg-yellow-400 p-3">
      <p className="text-yellow-900">
        We are currently using a free Polygon API key, which has a maximum limit
        of 5 requests per minute. If you see a 429 error, please wait a minute.
      </p>
    </div>
  );
};

export default GlobalAlert;
