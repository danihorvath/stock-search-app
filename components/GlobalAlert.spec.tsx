import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import GlobalAlert from "./GlobalAlert";

test("Snapshot Test", () => {
  const { container } = render(<GlobalAlert />);
  expect(container).toMatchSnapshot();
});
