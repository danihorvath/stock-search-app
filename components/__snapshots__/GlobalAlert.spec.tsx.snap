// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshot Test 1`] = `
<div>
  <div
    class="bg-yellow-400 p-3"
  >
    <p
      class="text-yellow-900"
    >
      This site is using a free Polygon API key, which has a maximum limit of 5 requests per minute. If you see a 429 error, please wait a minute.
    </p>
  </div>
</div>
`;
